api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto

```diff
@@ -101,4 +102,11 @@ message DnsCacheConfig {
   // ``envoy.restart_features.use_apple_api_for_dns_lookups`` runtime value is true during
   // server startup. Apple' API only uses UDP for DNS resolution.
   bool use_tcp_for_dns_lookups = 8;
+
+  // If DNS resolvers are specified,
+  // DNS cache will perform DNS resolution via those resolvers.
+  // Setting this value causes failure if the
+  // ``envoy.restart_features.use_apple_api_for_dns_lookups`` runtime value is true during
+  // server startup. Apple's API only allows overriding DNS resolvers via system settings.
+  repeated config.core.v3.Address dns_resolvers = 9;
```

Comment:
nit: Please add what will be used if not specified (i.e., using the default resolver).

Reply:
Thanks. Fixed the description.